cmake/CMakeBasics.cmake

```diff
@@ -12,7 +12,7 @@ option( ENABLE_TOTALVIEW_OUTPUT "" OFF )
 set( LVARRAY_BUILD_OBJ_LIBS OFF CACHE BOOL "" )
 
 
-# if( NOT BLT_CXX_STD STREQUAL c++14 )
+# if( NOT BLT_CXX_STD STREQUAL c++17 )
```

Comment:
I suppressed this on Frontier when I was swapping between standards to debug very Frontier / cce/hipcc specific issues. 

I think we should probably reactivate it to ensure that all our users / downstream host-configs appropriately update to C++17 when this is merged.

Reply:
yeah, I have just changed it locally. I ll push it later.